app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>"Karl's Most Epic Imaginary Ingredients Index"</h1>
+<% @ingredients.each do |ingredient| %>
+    <h2>Name: <%= ingredient.name %></h2>
+  <% end %>
+  <h3>Cost: <%= ingredient.cost %></h3>
```

Comment:
I don't think this last line is actually working because it is outside the `<% end %>` and therefore, you have lost the context of `ingredient`. Lines 4 and 5 need to be switched.